src/styles/modules/nav-list-iteam-text.scss

```diff
@@ -0,0 +1,19 @@
+.nav-list-item { 
+   margin: 0 58px 0 58px;
+   padding: 34px 0 34px 0;
+}
+
+li a:hover {
+   background: #ff3f40;
+   color: #e0e0e0;
+   text-decoration: none;
+   padding: 34px 0 34px 0;
+}
+
+a {
+   color: #b1b1b1;
+    text-decoration: none;
```

Comment:
remove extra space before `text-decoration`, and please for a future PRs re-review your PR on linter errors and spellings before create it

Reply:
done